dts/xtensa/intel/intel_adsp_ace15_mtpm.dtsi

```diff
@@ -36,7 +36,21 @@
 		};
 
 		power-states {
-			d0i3: idle {
+			clock_gating: idle1 {
+				compatible = "zephyr,power-state";
+				power-state-name = "active";
+				substate-id = <1>;
+				min-residency-us = <0>;
+				exit-latency-us = <0>;
+			};
+			clock_switching: idle2 {
+				compatible = "zephyr,power-state";
+				power-state-name = "active";
+				substate-id = <2>;
+				min-residency-us = <0>;
+				exit-latency-us = <0>;
+			};
```

Comment:
how these two states with the same residency time are supposed to be used ? The way it is now, only one will be selected ... unless it has a constraint

Reply:
I updated the description, please take a look

Reply:
this is still something that is not making sense for me. This is looking to me that you are creating this states just to set a property that you want to be used next time the cpu goes idle. These states are never used by the power management subsystem. The only usage is that you set a lock in one of them and check it when the system is goes idle (some state `!PM_STATE_ACTIVE`). It is much cleaner (and less costly) if you add and soc specific API since that is particular to this target (not even part of the SOF fw).